policyengine_us/parameters/gov/hhs/medicare/part_b/income_brackets/head_of_household.yaml

```diff
@@ -0,0 +1,35 @@
+description: Head of household filers use these income brackets for Medicare Part B premiums, based on income from the previous tax year.
```

Comment:
What is the amount value here? Is it the actual part B premium value. 

If so, can do describe it something like:
"HHS provides the following Medicaid Part B premium amounts for head of household filers, based on [income concept] from the previous tax year.